src/styles/blocks/calendar_day.scss

```diff
@@ -0,0 +1,53 @@
+@import "./utils/mixins";
+@import "./utils/variables";
+
+.calendar__day {
+  @include day-size($day-size);
+  background-color: #eee;
+  border: 1px solid black;
+  box-sizing: border-box;
+  display: flex;
+
+  &:first-child {
+    margin-left: 606px;
+  }
+
+  @for $day-number from 1 through 31 {
+    &:nth-child(#{$day-number}):before {
+ 
+      content: "#{$day-number}";
+      font-family: Arial, sans-serif;
+      color: black;
+      font-size: 30px;
+      margin: auto;
+    }
+  }
+
+  $days-of-week: 
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
+
+  @each $start-day, $start-day-value in $days-of-week {
+    &--start-day--#{$start-day}:first-child {
+      margin-left: $start-day-value * ($day-size + $gap-size);
+    }
+  }
+
+  @for $day from 28 through 30 {
```

Comment:
so, is it working with 30?

Reply:
no,calendar days change location 
![image](https://user-images.githubusercontent.com/93792500/205274846-ca2d2aa3-995b-49cf-8b52-d4a0ef3f025e.png)

Reply:
@vlad-demchuk

Reply:
no,it doesn't work in general